syntax = "proto3";

import "google/protobuf/timestamp.proto";

message Notification_proto {
    int32 id = 1;
    int32 user_id = 2;
    string message = 3;
    string type = 4;
    string status = 5;
    google.protobuf.Timestamp created_at = 6;
    google.protobuf.Timestamp updated_at = 7;
}
